distribution/bin/generate-binary-notice.py

```diff
@@ -81,11 +80,11 @@ def generate_notice(source_notice, dependences_yaml):
         parser.add_argument('out_path', metavar='<path to output file>', type=str)
         args = parser.parse_args()
 
-        with open(args.notice) as apache_notice_file:
+        with open(args.notice, encoding="ascii") as apache_notice_file:
```

Comment:
Just out of curiosity, is there any reason for this to be `ascii`?

Reply:
No, that is just how it was encoded originally I guess:
```
$ file -I licenses.yaml 
licenses.yaml: text/plain; charset=utf-8

$ file -I LICENSE
LICENSE: text/plain; charset=us-ascii

$ file -I NOTICE
NOTICE: text/plain; charset=us-ascii

$ file -I licenses/APACHE2 
licenses/APACHE2: text/plain; charset=us-ascii
```